akka-stream/src/main/scala/akka/stream/impl/fusing/StreamOfStreams.scala

```diff
@@ -9,15 +9,11 @@ import java.util.concurrent.atomic.AtomicReference
 
 import akka.NotUsed
 import akka.annotation.InternalApi
-import akka.stream.ActorAttributes.StreamSubscriptionTimeout
-import akka.stream.ActorAttributes.SupervisionStrategy
+import akka.stream.ActorAttributes.{ StreamSubscriptionTimeout, SupervisionStrategy }
```

Comment:
Nitpick: do we actually like that style more? Otherwise, better leave unchanged source files alone to avoid conflicts.

Reply:
We try to avoid these multi-imports since they easily cause git conflicts.

Reply:
(I don't think that should block merging this PR though)

Reply:
I think this is the result of idea's auto-imports, I'll revert it